test/unit_test.c

```diff
@@ -3369,8 +3368,13 @@ int main(void) {
   test_http_no_content_length();
   test_http_pipeline();
   test_http_range();
+#ifndef LOCALHOST_ONLY
   test_sntp();
   test_mqtt();
+  test_http_client();
+#else
+  (void) test_sntp, (void) test_mqtt, (void) test_http_client;
+#endif
```

Comment:
I guess this is a leftover from the Debian saga

Reply:
Ah, yeah